opm/test_util/EclFilesComparator.cpp

```diff
@@ -108,15 +108,15 @@ void ECLFilesComparator::deviationsForCell(double val1, double val2, const std::
         std::cout << "(value1, value2) = (" << val1 << ", " << val2 << ")\n\n";
     }
     if (val1 < 0) {
-        if (abs(val1) > absTolerance) {
+        if (std::abs(val1) > absTolerance) {
```

Comment:
@akva2 Indeed, it is the same on my machine. It always calls the int version. Weired.

For completeness, here is my testcase:
```c++
#include<cmath>
#include<cstdlib>
#include<iostream>
int main()
{
    double d=-1.5;
    std::cout<<"abs="<<abs(d)<<std::endl;
}
```
with output `abs=1`